### DISCUSSION TITLE
Service call buttons feature widget for tile cards

### DISCUSSION THREAD

**Discussion by lcavalli** [home-assistant/frontend#19205] [Dashboard] [answered]
Would you take into account a feature widget available for all tile cards with a button bar to call a service for each defined button. Number of buttons, icons, colours, services and parameters should be configurable. This will extend the usage of the available tile cards (e. g. I Need a button in my alarm tile card to trigger extra actions, like reset alarm memory).

**Comment by lcavalli** [comment]
It should be aesthetically equivalent to the cover open/close feature widget.

![image](https://github.com/home-assistant/frontend/assets/6123641/35dfd59c-71e3-41dc-b8ca-3bc88eb3cec3)

**Comment by BBE-FR** [comment]
I have a solution based on mushroom (almost the same look as Tiles) with the stack-in of a mushroom template card and a row of mushroom chips.

Described (in french) here: https://forum.hacf.fr/t/carte-mise-en-forme-de-ma-mushroom-room-card-et-tuto/20429

That is my temporary solution before using Tiles customizable features if it happens one day...

There is a simplified version with step by step tuto here: https://forum.hacf.fr/t/carte-mise-en-forme-de-ma-mushroom-room-card-et-tuto/20429/12

**Comment by BBE-FR** [comment, marked as answer]
There is this: 
https://github.com/Nerwyn/service-call-tile-feature
With the tile look. But it is only in YAML...

**Reply by BBE-FR** [reply]
This could be a nice improvement of an advanced tile card, with custom service call tile available through UI...
